docs/scripts/generateTemplateImages.js

```diff
@@ -84,10 +84,14 @@ function getTemplateList() {
 					deviceScaleFactor: 1,
 				});
 
-				// Scroll past the template banner
+				// Scroll past the header into the content area
+				// Note: We don't just use the `main` selector here as that can be different across templates
```

Comment:
Maybe we can look at this again when my PR gets in. We could export the templates without the surrounding Site layout and screenshot that.